CMakeLists.txt

```diff
@@ -44,15 +41,15 @@ include(PACE_Version)
 set(PACE_MLINT_IGNORE ${PACE_MLINT_IGNORE} "${Herbert_ROOT}/**/*.m")
 
 include(PACE_CodeAnalysis)
-if(${BUILD_TESTS})
-    include(CTest)
+include(CTest)
+if(BUILD_TESTING)
```

Comment:
Presuming that BUILD_TESTING is not required to be a $variable but checking - I can see it comes from the -D options below but so did $BUILD_TESTS

Reply:
If is an odd cookie and doesn't quire the expansion manually. Purely stylistic, but it's the way I usually see ifs. 
https://cmake.org/cmake/help/latest/command/if.html#variable-expansion